winch/codegen/Cargo.toml

```diff
@@ -17,6 +17,7 @@ target-lexicon = { workspace = true, features = ["std"] }
 # In the next iteration we'll factor out the common bits so that they can be consumed
 # by Cranelift and Winch.
 cranelift-codegen = { workspace = true }
+cranelift-wasm = { workspace = true }
```

Comment:
Ah no I don't think so, I originally didn't have the translation from the `WasmFuncType` to `FuncType` but since changed it to be more consistent with other Winch methods